README.md

```diff
@@ -496,6 +496,21 @@ deleteAll() async {
 
 ```
 
+## Environment Variables
+
+Environment variables are variables like those we create in our applications.
+They are dynamically named values in the operating system, which affect the behavior of the programs that consume it. They are stored in a "list" of key-values.
```

Comment:
Yes and No. This explanation is incomplete. We are better off adding more detail to our existing tutorial on this topic https://github.com/dwyl/learn-environment-variables and then _linking_ to it from `learn-flutter`.